mastodon_bot/core.cljs

```diff
@@ -138,12 +138,20 @@
       (when-not (:media-only? mastodon-config)
         (post-status text)))))
 
+(defn in [needle haystack]
+  (some (partial = needle) haystack))
+
+; If the text ends in a link to the media (which is uploaded anyway),
+; chop it off instead of including the link in the toot
+(defn chop-tail-media-url [text media]
+  (clojure.string/replace text #" (\S+)$" #(if (in (%1 1) (map :url media)) "" (%1 0))))
```

Comment:
This could just be `(string/replace ...)` since `clojure.string` is already aliased in the namespace.